ui/pages/send/send-footer/send-footer.container.js

```diff
@@ -88,7 +88,7 @@ function mapDispatchToProps(dispatch) {
         to,
       });
 
-      sendToken
+      return sendToken
```

Comment:
in the send-footer component we do a .then chain off of the sign method but we don't return anything from this function so its unclear to me how the .then() ever functioned before this.